.pre-commit-config.yaml

```diff
@@ -46,6 +46,9 @@ repos:
       - id: fix-byte-order-marker
       - id: check-case-conflict
       - id: check-toml
+      - id: file-contents-sorter
+        files: dictionary.*\.txt$
```

Comment:
May I ask you add the new files `*-additionnal.wordlist`? They live under `codespell_lib/tests/data` and are currently not sorted.

Reply:
Or even `*-additional.wordlist` as per #2976 !